skimage/util/tests/test_compare.py

```diff
@@ -1,27 +1,47 @@
 import numpy as np
-
-from skimage._shared.testing import assert_array_equal
-from skimage._shared import testing
+import pytest
 
 from skimage.util.compare import compare_images
 
 
-def test_compate_images_ValueError_shape():
+def test_compare_images_ValueError_shape():
     img1 = np.zeros((10, 10), dtype=np.uint8)
     img2 = np.zeros((10, 1), dtype=np.uint8)
-    with testing.raises(ValueError):
+    with pytest.raises(ValueError):
         compare_images(img1, img2)
 
 
+def test_compare_images_ValueError_args():
+    a = np.ones((10, 10)) * 3
+    b = np.zeros((10, 10))
+    with pytest.warns(FutureWarning):
+        with pytest.raises(ValueError):
+            compare_images(a, b, "unknown")
```

Comment:
```suggestion
      with pytest.raises(ValueError):
          compare_images(a, b, method="unknown")
```
I'd keep the tests focused and independent from each other; you're already testing the deprecation in another test. :)

Reply:
Ok, you're right. I'll just edit the indentation of your edit.